lib/nylas/draft.rb

```diff
@@ -39,9 +39,12 @@ class Draft
     transfer :api, to: %i[events files folder labels]
 
     def update(**data)
+      # If files are provided, replace files with file IDs
       self.files = data[:files] if data[:files]
       extract_file_ids!
       data[:file_ids] = file_ids
+      # If version is not provided, add version or request will fail
+      data[:version] = version unless data[:version]
```

Comment:
Could we add a test case for this in `draft_spec.rb`?

Reply:
Done, thank you for pointing that out!